ansible.cfg

```diff
@@ -9,3 +9,6 @@ connection_plugins = plugins/connection
 callback_plugins = plugins/callbacks
 transport = monkeypatched_ssh
```

Comment:
I would like to know a bit more on how pipelining is implemented and see CI results before merging.